test/mocks/Tempus.sol

```diff
@@ -59,4 +62,11 @@ contract Tempus {
 
         return depositAndFixReturn;
     }
+
+    function redeemToBacking(address o, uint256 m, uint256 a, address u) external {
```

Comment:
in methods that take more than 2 args we use a mapping whose first argument is the key whose value is map to hold the other args

https://github.com/Swivel-Finance/gost/blob/v2/test/mocks/Erc20.sol#L12=